common/constants.gradle

```diff
@@ -3,8 +3,8 @@ project.ext {
     projectName = "firebase-jobdispatcher-android"
     group = "com.firebase"
     version = "0.5.1"
-    buildtools = "25.0.0"
-    supportLibraryVersion = "25.0.0"
+    buildtools = "25.0.2"
+    supportLibraryVersion = "25.1.0"
```

Comment:
Do we need to bump these versions as well as include the `@Keep` annotation? I'd rather limit the change to the minimum of what's required to fix the issue.